### PR TITLE
String management

### PR DESCRIPTION
**Pull request by vidia**
Moved strings to static final references. There were too many raw references to Strings all over. This collects them into one place.

### CODE REVIEW

File: app/src/main/java/xyz/jhughes/laundry/adapters/MachineAdapter.java [hunk 1]
```diff
@@ -187,7 +189,7 @@ private void scheduleNotification(Notification notification, int delay) {
 
     private Notification getNotification(String content) {
         Notification.Builder builder = new Notification.Builder(c);
-        builder.setContentTitle("Purdue Laundry");
+        builder.setContentTitle("Purdue Laundry"); //TODO: Get string resource for App name.
```
Comment (vidia) [inline app/src/main/java/xyz/jhughes/laundry/adapters/MachineAdapter.java]: This should ideally be pulled out into a string reference.

File: app/src/main/java/xyz/jhughes/laundry/fragments/MachineFragment.java [hunk 2]
```diff
@@ -50,6 +50,7 @@
 
     private View rootView;
 
+    //TODO: What is this doing? What is special about these entries?
     public static String options = "Available|In use|Almost done|End of cycle";
```
Comment (vidia) [inline app/src/main/java/xyz/jhughes/laundry/fragments/MachineFragment.java:53]: This set of options is scattered around the code. It should be collected into one place. 

What is it doing @hughesjeff ? Did you write these parts?

**Review by vidia — COMMENTED** [review]
This PR isnt complete. It needs to fix a few things that I have commented in the review. 

EDIT: The PR is now complete.

File: app/src/main/java/xyz/jhughes/laundry/fragments/MachineFragment.java (hunk 2 above)
Comment (vidia) [reply app/src/main/java/xyz/jhughes/laundry/fragments/MachineFragment.java:53]: Fixed it. Its the filter options. I pulled them out, but it still works the same.

File: app/src/main/java/xyz/jhughes/laundry/LaundryParser/MachineTypes.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package xyz.jhughes.laundry.LaundryParser;
+
+/**
+ * Created by dmtsc on 10/18/2016.
+ */
+
+public class MachineTypes {
+    public static final String DRYER = "Dryer";
```
Comment (hughesjeff) [inline app/src/main/java/xyz/jhughes/laundry/LaundryParser/MachineTypes.java:8]: Should probably move these into strings.xml, but it works for now so we can get a release out.

File: app/src/main/java/xyz/jhughes/laundry/LaundryParser/MachineStates.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
+package xyz.jhughes.laundry.LaundryParser;
+
+/**
+ * Created by dmtsc on 10/18/2016.
+ */
+
+public class MachineStates {
+    public static final String IN_USE = "In use";
```
Comment (hughesjeff) [inline app/src/main/java/xyz/jhughes/laundry/LaundryParser/MachineStates.java:8]: Should probably move these into strings.xml, but it works for now so we can get a release out.

File: app/src/main/java/xyz/jhughes/laundry/LaundryParser/Rooms.java [hunk 5]
```diff
@@ -28,10 +28,10 @@ private Rooms() {
             roomsToImage.put(room, toImageResourceId(room));
         }
 
-        machineAvailabilityColors.put("Available", R.color.Available);
-        machineAvailabilityColors.put("In use", R.color.InUse);
-        machineAvailabilityColors.put("Almost done", R.color.AlmostDone);
-        machineAvailabilityColors.put("End of cycle", R.color.Finished);
+        machineAvailabilityColors.put(MachineStates.AVAILABLE, R.color.Available);
```
Comment (hughesjeff) [inline app/src/main/java/xyz/jhughes/laundry/LaundryParser/Rooms.java:31]: Literally what strings.xml was made for :P

File: app/src/main/java/xyz/jhughes/laundry/fragments/MachineFragment.java [hunk 6]
```diff
@@ -50,8 +51,6 @@
 
     private View rootView;
 
-    public static String options = "Available|In use|Almost done|End of cycle";
```
Comment (hughesjeff) [inline app/src/main/java/xyz/jhughes/laundry/fragments/MachineFragment.java:52]: This was used... Won't compile without.

**Comment by hughesjeff** [comment]
Doesn't compile. Please fix  last comment.

**Review by hughesjeff — CHANGES_REQUESTED** [review]

File: app/src/main/java/xyz/jhughes/laundry/LaundryParser/MachineTypes.java (hunk 3 above)
Comment (vidia) [reply app/src/main/java/xyz/jhughes/laundry/LaundryParser/MachineTypes.java:8]: I really do want to have this in Strings.xml. The issue is that there isnt always a context available when we want access to these. 

I also want an overhaul of things a bit .

File: app/src/main/java/xyz/jhughes/laundry/LaundryParser/Rooms.java (hunk 5 above)
Comment (vidia) [reply app/src/main/java/xyz/jhughes/laundry/LaundryParser/Rooms.java:31]: Yeah, But again the lack of a context in all relevant places makes it rough. We may need to init these in Activity.onCreate() and then access statically like this. But even that isnt COMPLETELY ideal.

**Pull request merged**